lib/export/formatters/dataElementFormatterCimpl6.js

```diff
@@ -0,0 +1,688 @@
+            subNs = newDe.fields[j].identifier.namespace;
+            added = true;
+            break;
+          }
+          else if (splitFormattedValue[i + 1].trim() === newDe.fields[j].effectiveIdentifier.name) {
+            modifiedFormattedValue += `.${splitFormattedValue[i + 1]}`;
+            subNs = newDe.fields[j].effectiveIdentifier.namespace;
+            added = true;
+            break;
+          }
+        }
+        if (!added) {
+          if (newDe.value) {
+            if (newDe.value.options) {
+              let a = 0;
+              for(a = 0; a < newDe.value.options.length; a++) {
+                if (splitFormattedValue[i +1].trim() === newDe.value.options[a].identifier.name) {
+                  modifiedFormattedValue += `[${splitFormattedValue[i + 1]}]`;
+                  subNs = newDe.value.options[a].identifier.namespace;
+                }
+                else if (splitFormattedValue[i +1].trim() === newDe.value.options[a].effectiveIdentifier.name) {
+                  modifiedFormattedValue += `[${splitFormattedValue[i + 1]}]`;
+                  subNs = newDe.value.options[a].effectiveIdentifier.namespace;
+                }
+              }
+            }
+            else {
+              if (splitFormattedValue[i + 1].trim() === newDe.value.identifier.name) {
+                modifiedFormattedValue += `[${splitFormattedValue[i + 1]}]`;
+                subNs = newDe.value.identifier.namespace;
+              }
+              else if (splitFormattedValue[i + 1].trim() === newDe.value.effectiveIdentifier.name) {
+                modifiedFormattedValue += `[${splitFormattedValue[i + 1]}]`;
+                subNs = newDe.value.effectiveIdentifier.namespace;
+              }
+            }
+          }
+        }
+        newDe = this.specs.dataElements.find(subNs, subName);
+      }
+
+      formattedValue = modifiedFormattedValue;
+    }
+
+    // then try the mess that is constraints
+    if ('constraints' in value && value.constraints.length > 0) {
+      if (!formatAsField && value.constructor.name == 'RefValue' && value.constraints.length == 1 && value.constraints[0].constructor.name == 'TypeConstraint') {
+        formatAsField = true;
+        formattedValue = `${rightPad('',this.maxRightPad)}Value`;
+      }
+      formattedValue = this.formatValueWithConstraints(formattedValue, value, de, formatAsField);
+      formattedValue = this.replaceValue(formattedValue);
+      if (!formatAsField && value.inheritance === 'overridden') {
+        if (de.identifier.name === value.constraints[0].lastModifiedBy.name) {
+          let formattedValueParts = formattedValue.split(" ");
+          let firstPart = `${rightPad('',this.maxPlacement)}Value only ${formattedValueParts[0]}\n`
+          if (formattedValueParts.length > 1) {
+            firstPart += `${rightPad('',this.maxPlacement)}Value ${formattedValueParts.slice(1).join(" ")}`;
+          }
+          return firstPart.replace(/[0-9]+..[0-9]+/, '');
```

Comment:
Yes, it should be okay because we have the rule that for Value fields, we remove the cardinality. I'm only removing the cardinality for those instances.